### ISSUE TITLE
Support for getting Elasticsearch info

### ISSUE THREAD

**Issue by calexandre** [nosinovacao/name-sdk#19]
Currently, NAME does not support elasticsearch integration.

In order to obtain elasticsearch information, one should perform the following query:

```
# curl http://elasticsearch:9200
{
  "name" : "a5045f503f93",
  "cluster_name" : "use",
  "cluster_uuid" : "Czl7TeOvTEe0-8sXhiKUNw",
  "version" : {
    "number" : "5.5.1",
    "build_hash" : "19c13d0",
    "build_date" : "2017-07-18T20:44:24.823Z",
    "build_snapshot" : false,
    "lucene_version" : "6.6.0"
  },
  "tagline" : "You Know, for Search"
}
```

**Comment by Symbianx** [comment]
@Mozcatel will take care of this.

**Comment by Symbianx** [comment]
Merged into dev, fixed by #21.
Thanks @Mozcatel.
